src/lang.envd

```diff
@@ -1,4 +1,9 @@
 def install_golang(version="1.18.8"):
```

Comment:
Is it better to be `golang` instead of `install_golang`, since other functions here don't contain verbs

Reply:
SGTM. Fixed.